app/parse.py

```diff
@@ -8,8 +16,43 @@ class Quote:
     tags: list[str]
 
 
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag_soup.text for tag_soup in quote_soup.select(".tags > a")]
+    )
+
+
+def get_page_quotes(url: str, quotes: list[Quote]) -> None:
```

Comment:
are u sure for type annotation for quotes ? does it passes with quotes ?

Reply:
yes